api/v1alpha5/virtualmachine_affinity_types.go

```diff
@@ -128,12 +74,12 @@ type VMAffinityTerm struct {
 	// Please note, The following rules apply when specifying the topology key in the context of a zone/host.
 	//
 	// - When topology key is in the context of a zone, the only supported verbs are
-	//   PreferredDuringSchedulingIgnoredDuringExecution and RequiredDuringSchedulingIgnoredDuringExecution.
+	//   PreferredDuringSchedulingPreferredDuringExecution and RequiredDuringSchedulingPreferredDuringExecution.
 	// - When topology key is in the context of a host, the only supported verbs are
```

Comment:
Both of these points are the same, no?
They are talking about VM-VM AAF with hosts.  And for that we support RequiredPreferred (mobility operator) and PreferredPreferred (VKS and regular users).